waypoint_horizon: 10
reach_waypoint_threshold: 0.2